apps/console/src/extensions/configs/models/common.ts

```diff
@@ -41,9 +41,12 @@ export interface CommonConfig {
         getHeaderSubPanelExtensions: () => HeaderSubPanelItemInterface[];
         /**
          * Get the user dropdown link extensions.
+         * @param {string} tenantDomain - Current tenant
```

Comment:
Missing new line here between method description and parameter descriptions.
Shall we add a `.` at the end?

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/3122/commits/6924afc4f3666ebe765dacebb9bb90f418354c61